front/src/components/Index.jsx

```diff
@@ -1,21 +1,24 @@
 import { Link } from "react-router-dom";
 import React from "react";
 import useQuestions from "components/Questions";
-// Currently displays all questions in database. Not practical if there are a lot of questions. Need to update so
-// that it displays the five latest polls.
+
 const Index = () => {
   const questions = useQuestions();
   const listQuestion = (item) => {
     const pollHeader = `- ${item.text}`;
-    const publishDate = `Published: ${item.pubDate}`;
+    //const publishDate = `Published: ${item.pubDate}`;
```

Comment:
We shouldn't ever check skeleton code into master if we can help it. Should we delete this?

Reply:
Agree. Will delete.